.travis.yml

```diff
@@ -3,6 +3,7 @@ before_script:
 - composer install --dev --no-interaction
 - cd test
 script:
+- "php ../vendor/bin/phpcs -n --exclude=PEAR.Commenting.FileComment,PEAR.Commenting.ClassComment,PEAR.Commenting.FunctionComment ../lib/"
```

Comment:
How about using the following `phpcs` command to check the `PSR-2` coding style?

```php
./vendor/bin/phpcs -n --standard=PSR2 lib/ test/
```

I think this command I suggest is good enough to use :).

Reply:
Thank you for review but In [CONTRIBUTING.md](https://github.com/sendgrid/smtpapi-php/blob/master/CONTRIBUTING.md#style-guidelines-and-naming-conventions) it's using PEAR standard. I will add /test in script.

Reply:
Mm....The coding standard is different from [sendgrid-php](https://github.com/sendgrid/sendgrid-php/blob/master/CONTRIBUTING.md). I think it also uses the PSR-2 coding style.

If this repo uses `PEAR` coding style, this comment I write will not be concerned :).